src/literal/teddy_avx2/imp.rs

```diff
@@ -461,12 +462,20 @@ impl Mask {
         let byte_lo = (byte & 0xF) as usize;
         let byte_hi = (byte >> 4) as usize;
 
-        let lo = self.lo.extract(byte_lo) | ((1 << bucket) as u8);
-        self.lo.replace(byte_lo, lo);
-        self.lo.replace(byte_lo + 16, lo);
```

Comment:
@RReverser for example, see here. Every call to replace requires spilling the vector to memory, modifying its contents, and then writing it back to a SIMD register.

The change explicitly spills the vector to the stack once. Modifies the stack, and moves the memory back into the vector, once.